repository/RediStick-Connection-Pool.package/RsRedisConnectionPool.class/instance/newRedisConnectionAt..st

```diff
@@ -1,3 +1,9 @@
 factory
-newRedisConnectionAt: key
-	^self redisWrapperClass on: (self createStickAt: key)
\ No newline at end of file
+newRedisConnectionAt: aKey
+	| redisWrapper |
+	redisWrapper := self redisWrapperClass on: (self createStickAt: aKey).
+	connections
```

Comment:
It may be confusing, but `connections` originally assumes a dictionary of key -> redisWrappers, not key -> redisWrapper.

Pseudo code:
{
connectionKeyA -> {a RsRedisWrapper. a RsRedisWrapper. a RsRedisWrapper...} (not exceeding #maxConnectionsPerKey).
connectionKeyB -> {a RsRedisWrapper. a RsRedisWrapper. a RsRedisWrapper...} (not exceeding #maxConnectionsPerKey).
}

Please see getConnectionAt:. It registers a newly created connection(wrapper) to the connections for each key. 
https://github.com/mumez/RediStick/blob/5f1539dd661abc1d57c4b7a2d0f007869e9f741c/repository/RediStick-Connection-Pool.package/RsRedisConnectionPool.class/instance/getConnectionAt..st

Reply:
ok so the key what is it ? why we have an association between a key and a collection of RedisWrapper , maybe I'm confusing I never use/create ConnectionPool